internal/development.md

```
@@ -386,19 +386,19 @@ RAGFlow(user)> ocr with 'paddleocr-vl-0.9b@test@baidu' file './internal/text.jpg
 
 ### 6.26 Chunk Management Commands
 
-- Create a dataset table with vector size
+- Create a chunk store with vector size
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)